activemodel/lib/active_model/attribute_assignment.rb

```diff
@@ -2,8 +2,15 @@
 
 module ActiveModel
   module AttributeAssignment
+    extend ActiveSupport::Concern
+
     include ActiveModel::ForbiddenAttributesProtection
 
+    # Alias for `assign_attributes`.
+    def attributes=(attributes)
```

Comment:
is related to multiparameters? i remember there was some concerns in tying this alias with rest of attribute assignment because some custom models may define an attribute called `attributes`